### PR TITLE
Add kube-up functionality

### PR DESCRIPTION
**Pull request by zreigz**

### CODE REVIEW

**Comment by zreigz** [comment]
@cheld @batikanu @f-higashi @taimir @kenan435 
Please review

File: util.sh [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        elif [ $status = "CREATE_FAILED" ]; then
+          echo "Cluster not created. Please check stack logs to find the problem"
+          break
+        fi
+     else
+       echo "Cluster not created. Please verify if process started correctly"
+       break
+     fi
+     sleep 30
+  done
```
Comment (f-higashi) [inline util.sh:62]: Why sleep 30?  
I think it is better to use SECONDS variable
http://linux.die.net/man/1/bash

in first of script, set `SECONDS=0`  and

```
while (( $SECONDS < ${STACK_CREATE_TIMEOUT}*60 )) ;do
   ----
```

Other part is LGTM

Comment (batikanu) [inline util.sh:62]: LGTM

Comment (zreigz) [inline util.sh:62]: Thanks. Good to know about SECONDS variable. I additionally added spinner to show progress and left sleep for 1 second to delay checking status

**Comment by zreigz** [comment]
PTAL

**Comment by f-higashi** [comment]
LGTM
:+1:

**Pull request merged**